op.go

```diff
@@ -229,16 +227,16 @@ func updateStatement(kn, cfName string, fields map[string]interface{}, opts Opti
 	buf := new(bytes.Buffer)
 	buf.WriteString(fmt.Sprintf("UPDATE %s.%s ", kn, cfName))
 
+	ret := []interface{}{}
```

Comment:
Given you seem to add strings to this, would it makes sense to type it as `[]string` instead?

Reply:
It needs to be returned as `[]interface{}{}`. We do add strings but you can add any type